Source/WebKit/UIProcess/Cocoa/ProcessAssertionCocoa.mm

```diff
@@ -378,14 +378,18 @@ static ASCIILiteral runningBoardDomainForAssertionType(ProcessAssertionType asse
         auto didInvalidateBlock = [weakThis = ThreadSafeWeakPtr { *this }, runningBoardAssertionName] () {
             auto strongThis = weakThis.get();
             RELEASE_LOG(ProcessSuspension, "%p - ProcessAssertion: RBS %{public}s assertion for process with PID=%d was invalidated", strongThis.get(), runningBoardAssertionName.characters(), strongThis ? strongThis->m_pid : 0);
-            if (strongThis)
-                strongThis->processAssertionWasInvalidated();
+            RunLoop::main().dispatch([weakThis = weakThis] {
```

Comment:
`weakThis = WTFMove(weakThis)`? Would just need to make the outer lambda `mutable`. Ditto for `willInvalidateBlock`.